addons/binding/org.openhab.binding.pentair/src/main/java/org/openhab/binding/pentair/handler/PentairEasyTouchHandler.java

```diff
@@ -0,0 +1,426 @@
+                (byte) circuit, (byte) ((state) ? 1 : 0) };
+
+        PentairPacket p = new PentairPacket(packet);
+
+        PentairBaseBridgeHandler bbh = (PentairBaseBridgeHandler) this.getBridge().getHandler();
+        bbh.writePacket(p);
+    }
+
+    /**
+     * Method to set heat point for pool (true) of spa (false)
+     *
+     * @param Pool pool=true, spa=false
+     * @param temp
+     */
+    public void setPoint(boolean Pool, int temp) {
+        // [16,34,136,4,POOL HEAT Temp,SPA HEAT Temp,Heat Mode,0,2,56]
+        // [165, preambleByte, 16, 34, 136, 4, currentHeat.poolSetPoint, parseInt(req.params.temp), updateHeatMode, 0]
+        int spaset = (!Pool) ? temp : phspcur.spasetpoint;
+        int poolset = (Pool) ? temp : phspcur.poolsetpoint;
+        int heatmode = (phspcur.spaheatmode << 2) | phspcur.poolheatmode;
+
+        byte[] packet = { (byte) 0xA5, (byte) 0x01, (byte) m_id, (byte) 0x00 /* source */, (byte) 0x88, (byte) 0x04,
+                (byte) poolset, (byte) spaset, (byte) heatmode, (byte) 0 };
+
+        logger.info("Set {} temperature: {}", (Pool) ? "Pool" : "Spa", temp);
+
+        PentairPacket p = new PentairPacket(packet);
+
+        PentairBaseBridgeHandler bbh = (PentairBaseBridgeHandler) this.getBridge().getHandler();
+        bbh.writePacket(p);
+    }
+
+    @Override
+    public void processPacketFrom(PentairPacket p) {
+        switch (p.getAction()) {
+            case 1: // Write command to pump
+                logger.trace("Write command to pump (unimplemented): {}", p.toString());
+                break;
+            case 2:
+                if (p.getLength() != 29) {
+                    logger.debug("Expected length of 29: ", p.toString());
+                    return;
+                }
+
+                PentairPacket_Status p29Old = p29cur;
+                p29cur = new PentairPacket_Status(p);
+
+                updateChannel(EASYTOUCH_POOL, p29Old);
+                updateChannel(EASYTOUCH_POOLTEMP, p29Old);
+                updateChannel(EASYTOUCH_SPATEMP, p29Old);
+                updateChannel(EASYTOUCH_AIRTEMP, p29Old);
+                updateChannel(EASYTOUCH_SOLARTEMP, p29Old);
+                updateChannel(EASYTOUCH_HEATACTIVE, p29Old);
+                updateChannel(EASYTOUCH_POOL, p29Old);
+                updateChannel(EASYTOUCH_SPA, p29Old);
+                updateChannel(EASYTOUCH_AUX1, p29Old);
+                updateChannel(EASYTOUCH_AUX2, p29Old);
+                updateChannel(EASYTOUCH_AUX3, p29Old);
+                updateChannel(EASYTOUCH_AUX4, p29Old);
+                updateChannel(EASYTOUCH_AUX5, p29Old);
+                updateChannel(EASYTOUCH_AUX6, p29Old);
+                updateChannel(EASYTOUCH_AUX7, p29Old);
+                updateChannel(DIAG, p29Old);
+
+                break;
+            case 4: // Pump control panel on/off
+                logger.trace("Turn pump control panel {}: {}", p.getDest(), p.getByte(PentairPacket.STARTOFDATA));
+
```

Comment:
Added a comment - i kept these here to document these commands and for future enhancement.